README.md

```diff
@@ -1 +1,143 @@
-# capstone-project-1
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+   
+  <br/>
+
+  <h3><b>My portfolio</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 portfolio <a name="about-project"></a>
+This pull is created to add javascript to my first capstone project and make interaction on my menu bars and times 
+This PR updates the responsive desktop design for our website to improve its usability on larger screens. The goal of this update is to provide a better user experience for desktop users and ensure that our website looks great on all devices
+
+## 🛠 Built With <a name="built-with"></a>
+<ul>
+  <li>HTML5</li> 
+  <li>CSS3</li>
+  <li>jJavaScript</li>
+  <li>flexbox</li>
+  <li>CSS Grid</li>
+</ul>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="#">n/a</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="#">n/a</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- Updated the HTML and CSS files to improve the responsiveness of the website on larger screens.
+- Implemented a new layout technique to better optimize the website for desktop screens.
+- Improved the typography and spacing of the website to enhance its visual appeal.
+- I also used CSS media queries to adjust the layout of the website based on the screen size.
+- I Used basic JavaScript syntax.
+- I Used JavaScript to manipulate DOM elements.
+- I Used JavaScript events.
+ -I used objects to store and access data.
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+- vs code 
+- github
+- laptop
+
+### Setup
+
+Clone this repository to your desired folder:
+
+  - cd my-folder
+  - git clone https://github.com/massazoum/capstone-project-1.git
+  
+  - git clone
+  - npm install or npm init -y
+
+### Run tests
+
+This website has been completed and deployed
+
+### Deployment
+
+  The changes will be deployed to the website through our standard deployment process.
+- [PortFolio](https://massazoum.github.io/Portfolio-setup-and-mobile-first/)
```

Comment:
- [ ] Kindly use the correct link for the live pages as it is professional and easy to view your project.